File: Baekjoon/1단계 입출력과 사칙연산/python/곱셈.py
import sys

a = int(sys.stdin.readline().strip())
b = int(sys.stdin.readline().strip())

digitOne = (b % 10)
digitTen = (b % 100 // 10)
digitHundred = b // 100

print(a * digitOne)
print(a * digitTen)
print(a * digitHundred)
print(a * b)

File: Baekjoon/java/src/1단계/곱셈.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.StringTokenizer;

public class Main{
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int a = Integer.parseInt(br.readLine());
        int b = Integer.parseInt(br.readLine());
        int digitOne = a * (b % 10);
        int digitTen = a * (b % 100 - b % 10);
        int digitHundred = a * (b % 1000 - b % 100);
        System.out.println(digitOne);
        System.out.println(digitTen / 10);
        System.out.println(digitHundred / 100);
        System.out.println(digitOne + digitTen + digitHundred);
    }
}